lib/dialects/postgres/query.js

```diff
@@ -95,8 +95,8 @@ class Query extends AbstractQuery {
         return queryResult;
       })
       .then(queryResult => {
-        const rows = queryResult.rows;
-        const rowCount = queryResult.rowCount;
+        const rows = Array.isArray(queryResult) ? queryResult.reduce((rows, r) => rows.concat(r.rows), []) : queryResult.rows;
+        const rowCount = Array.isArray(queryResult) ? queryResult[0].rowCount : queryResult.rowCount;
```

Comment:
shouldn't this be `queryResult.map(row => row.rowCount).reduce((prev, next) => prev + next)` or so? 
In your example, both results had rowCount 1, so in sum they would be 2, which seems to be correct.